ssl/statem/statem.c

```diff
@@ -8,6 +8,7 @@
  */
 
 #include <openssl/rand.h>
+#include "e_os.h"
```

Comment:
This one also seems to be dependency on (not-really-os-specific) OPENSSL_CONF...